### DISCUSSION TITLE
Datasets to constraint eligible areas

### DISCUSSION THREAD

**Discussion by pz-max** [pypsa-meets-earth/pypsa-earth#83] [Ideas]
A list of datasets that can be applied for constraining the eligible area of energy assets (wind, solar, etc.):

- **WPDA Dataset:** World Database on Protected Areas. UNEP-WCMC and IUCN (2021), Protected Planet: The World Database on Protected Areas (WDPA) and World Database on Other Effective Area-based Conservation Measures (WD-OECM) [Online], August 2021, Cambridge, UK: UNEP-WCMC and IUCN. Available at: www.protectedplanet.net.
- **GEBCO data:** A [bathymetric](https://en.wikipedia.org/wiki/Bathymetry) dataset with a global terrain model for ocean and land at 15 arc-second intervals by the [General Bathymetric Chart of the Oceans (GEBCO)](https://www.gebco.net/data_and_products/gridded_bathymetry_data). It can help to constrain eligible offshore areas and build assets in mountainous areas relevant to certain technologies.
- **Copernicus Dataset:** Copernicus Global Land Service offers openly land coverage data between 2015-2019 [here](https://land.copernicus.eu/global/). The [data explorer](https://lcviewer.vito.be/2015) gives a good overview of available land coverage data. This data might be in particular interesting to exclude closed forest areas and built-up areas.
- **NatureMapEarth data:** A collection of various 'nature' datasets which might be useful in many aspects. Here their [data explorer](https://explorer.naturemap.earth/map) and [transparently recorded data methods](https://explorer.naturemap.earth/methods)
- **Building data:** Google just shared in 2021 a  study "Continental-Scale Building Detection from High-Resolution Satellite Imagery" that might be useful to constraining the eligible area https://arxiv.org/abs/2107.12283 . OpenStreetMap offers another dataset for that purpose which could be assessed by the "Google" Satellite approach.

Thanks @jarry7 for your contribution.

**Comment by euronion** [comment]
Also of relevance: Population density in order to restrict utility scale RES projects in highly populated areas.

See: https://github.com/pypsa-meets-africa/pypsa-africa/discussions/46#discussioncomment-1123620

**Reply by pz-max** [reply]
Absolutely, missed that one!

**Reply by pz-max** [reply]
Assuming that people live in houses, also rooftop recognition data might be useful. 
Added to the list above great work from Google.

**Reply by euronion** [reply]
Rooftop recognition (also for other types of buildings other than residential buildings) in general is of some relevance! Let's first focus on utility scale projects :)

**Reply by jarry7** [reply]
I would tend to agree here with @euronion but it is a real challenge as more and more distributed/embedded generation capacity is deployed (for future proofing the approach).

I worry more about the required resolution (and related cost) of updated satellite data and not so much on whether the ML approaches can do the image recognition. Just a note.

**Reply by pz-max** [reply]
@jarry7 @euronion I just meant rooftop area data:
- For defining eligible areas for utility-scale generation capacity expansion. I guess rooftop areas and an offset around these areas might serve well for that purpose. Maybe even better than using population raster methods only. 

But yes absolutely, the rooftop data can also help to quantify i.e. the rooftop solar potentials etc. 

For the prototype, we can keep things simple implementing only low number of datasets and see how we are doing with time.

**Reply by pz-max** [reply]
@jarry7 Yes we can detect everything with satellites (I think we can even give voltage level estimations for substations with HQ resolution images). It is just a matter of costs regarding the resolution. Let's see how much money our AI team (@LukasFrankenQ ) requires on detecting HV network assets.
